tools/unsupervised_dataset/run_crawler.py

```diff
@@ -1,11 +1,16 @@
 """ Script to initialize the SQL crawler on a website of the user's choice """
 
 import sys
+import argparse
 from sql_crawler import crawler
 
 def start_crawler():
-    urls = sys.argv[1:]
-    new_crawler = crawler.Crawler(urls, max_size=50)
+    parser = argparse.ArgumentParser(description="SQL Web Crawler")
+    parser.add_argument("urls", help="A list of URLs to be crawled", nargs='+')
```

Comment:
Is the list separated by comma or something? maybe add that as well in description